pygmt/base_plotting.py

```diff
@@ -1918,3 +1918,197 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        A="vector",
+        B="frame",
+        C="cmap",
+        D="rescale",
+        E="uncertainty_color",
+        G="facecolor",
+        J="projection",
+        L="line",
+        N="no_clip",
+        R="region",
+        S="scaling",
+        U="timestamp",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
```

Comment:
```suggestion
        Y="yshift",
        p="perspective",
        t="transparency",
```
Add extra aliases for 3D **p**erspective plots and **t**ransparaency.